snippy_tseemann/Dockerfile

```diff
@@ -16,8 +16,7 @@ RUN cpanm Bio::Perl
 # Java
 RUN apt-get install -y default-jre
 # For Linuxbrew
-RUN apt-get install -y build-essential curl file git python-setuptools ruby locales
```

Comment:
may be you should separete this in different lines to simplify building process.